MKSOutputDevice.py

```diff
@@ -882,6 +882,121 @@ def resumePrint(self):
             self._ischanging = True
         self._sendCommand("M24")
 
+    def printer_set_connect(self):
+        self._sendCommand("M20")
+        self.setConnectionState(cast(ConnectionState, UnifiedConnectionState.Connected))
+        self.setConnectionText(self._translations.get("connected"))
+
+    def get_current_temp(self, temp):
+        return float(temp[0:temp.find("/")])
+
+    def get_target_temp(self, temp):
+        return float(temp[temp.find("/") + 1:len(temp)])
+
+    def printer_info_update(self, info):
+        printer = self.printers[0]
+        t0_temp = info[info.find("T0:") + len("T0:"):info.find("T1:")]
+        t1_temp = info[info.find("T1:") + len("T1:"):info.find("@:")]
+        bed_temp = info[info.find("B:") + len("B:"):info.find("T0:")]
```

Comment:
I'd suggest moving this `info.find("T0:") + len("T0:"):info.find("T1:")` as short method, because logic of this lines (898-900 ) are the same.